tests/test_clan_war_players.py

```diff
@@ -0,0 +1,28 @@
+
+import pytest
+import unittest
+from mockito import when
+from clash_royale_service import ClashRoyaleService
+from tests.resources import clash_royale_client_currentriverrace, clash_royale_client_responses
+
+
+class TestClanWarPlayers(unittest.TestCase):
+    def test_clan_players_remaining_war_attacks(self):
+        service = ClashRoyaleService()
+        clan_tag = "#9GULPJ9L"
+
+        # Mock API responses and certain function calls
+        when(service.clash_royale_client).get_current_river_race(clan_tag).thenReturn(
+            clash_royale_client_currentriverrace.CURRENT_RIVER_RACE_API_RESPONSE
+        )
+        when(service.clash_royale_client).get_clan_info(clan_tag).thenReturn(
+            clash_royale_client_responses.CLAN_INFO_API_RESPONSE
+        )
+
+        all_current_war_players_output = service.clan_players_remaining_war_attacks(clan_tag)
+        # Note 1: there are three players participated in the war, one of them completed all attacks and hence the name is not listed
+        # Note 2: two players have not completed all the war attacks, hence there are two entries in the list
+        player_1 = all_current_war_players_output[0]
+        player_2 = all_current_war_players_output[1]
+        self.assertEqual(len(all_current_war_players_output),2)
+        self.assertTrue(player_1.decks_used_today >= player_2.decks_used_today)
```

Comment:
Lmao, shouldn't you be doing asserts on the exact value of decks used by a specific player is equal to an expected value instead of comparing player_1 deckUsedCount with player_2 deckUsedCount?.....

I don't see the point of 
```python
self.assertTrue(player_1.decks_used_today >= player_2.decks_used_today)
```
Your code could be spitting out the wrong decksUsed counts for player1 and player2 and this assert wouldn't detect that

Reply:
True 😂

Reply:
I'll add the test case for expected value, this assert will remain as im also testing for sorting part of the list.